nanome/_internal/_structure/_complex.py

```diff
@@ -21,6 +23,7 @@ def __init__(self):
         self._selected = False #selected on live
         self._surface_dirty = False
         self._surface_refresh_rate = -1.0  # Not used yet, future auto surface refresh
+        self._box_label = ""
```

Comment:
Everything related to box_label is unrelated to this feature. But I have to get it in with this PR, since it changed the complex serializer on nanome-core, but wasn't merged in nanome-lib